### DISCUSSION TITLE
Hello, I am getting an error when installing utm on my MAC BOOK AIR M2 device. how can you help

### DISCUSSION THREAD

**Discussion by Yilmazogluu** [utmapp/UTM#6641] [UTM Help (macOS)] [answered]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
Replace this text with a clear and concise description of what the issue is.

**Configuration**  
* UTM Version: 
* macOS Version: 
* Mac Chip (Intel, M1, ...): 

**Crash log**  
If the app crashed, you need a crash log. To get your crash log, open Console.app, go to `Crash Reports`, and find the latest entry for either UTM, QEMU, QEMUHelper, or qemu-\*. Right click and choose `Reveal in Finder`. Attach the report here.

**Debug log**  
For all issues related to running a VM, _including_ crashes, you should attach a debug log. (This is unavailable for macOS-on-macOS VMs. Attach an excerpt of your system log instead.)
To get the Debug log: open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**  
If your issue is related to a specific VM, please upload the config.plist from your VM's .utm directory. To get this, right-click the VM in UTM and choose "Show in Finder". Then right-click the selected file in Finder and choose "Show Package Contents". The config.plist file is now visible. Right-click it and choose "Compress". Attach the resulting config.plist.zip file here.
You can upload the entire .utm if needed, but note this includes your VM's drive image and may contain personal data. Since Github has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by Yilmazogluu** [comment]
Hello, I am getting an error when installing utm on my MAC BOOK AIR M2 device. How can you help?

I accidentally purchased UTM twice, can you help me with the return?

**Comment by Yilmazogluu** [comment]
<img width="756" alt="Ekran Resmi 2024-08-30 01 43 30" src="https://github.com/user-attachments/assets/c2b2a5e6-0209-4990-b6a0-2745c8a74d03">

error

**Reply by osy** [reply, marked as answer]
You didn't press any key on startup to enter setup

**Reply by Yilmazogluu** [reply]
Basıyorum ama yine aynı hatayı alıyorum

**Reply by Yilmazogluu** [reply]
Lütfen yardım edin kafayı yiyeceğim

**Comment by Yilmazogluu** [comment]
[config.plist.zip](https://github.com/user-attachments/files/16815076/config.plist.zip)

config listesini yükledim inceleyip yardımcı olabilirseniz sevinirim

**Comment by Yilmazogluu** [comment]
Sizin geliştirdiğiniz UTM programına tüküreyim, hiçbir sanal makineyi yüklemedi, burdan yazmama rağmen bir arkadaşımın haricinde kimse dönmedi,   bir değil iki UTM parası ödedim geliştiricilere zehir zıkkım olsun

**Comment by Yilmazogluu** [comment]
Yanlışlıkla UTM iki defa satınalınmıştır. birinin üçretini nasıl tahsil edebilirim

**Comment by Yilmazogluu** [comment]
UTM was accidentally purchased from the App Store twice.  I request the return of one of them. How can you help?
